src/app/components/Modals/Privacy/data/privacy.tsx

```diff
@@ -0,0 +1,696 @@
+export const Privacy = () => {
+  return (
+    <div className="rounded-2xl border border-neutral-content p-6 dark:border-neutral-content/20 overflow-y-scroll no-scrollbar max-h-[100vh] terms-privacy-container">
+      <p className="italic">Effective as of 20 August 2024</p>
+      <br />
+      <p>
+        Babylon Labs Ltd (<strong>&quot;Labs&quot;</strong>,{" "}
+        <strong>&quot;we&quot;</strong>, <strong>&quot;us&quot;</strong> or{" "}
+        <strong>&quot;our&quot;</strong>) is committed to protecting the privacy
+        of our users, stakeholders, and business partners. This Privacy Policy
+        describes how we use, disclose, and safeguard any personal information
+        that we collect through our digital or online properties or services
+        (including as applicable, our website, social media pages, marketing
+        activities, live events and other activities described in this Privacy
+        Policy (each separately and collectively, the{" "}
+        <strong>&quot;Service&quot;</strong>).
+      </p>
+      <br />
+      <p>
+        This Privacy Policy takes into account the requirements of the British
+        Virgin Islands Data Protection Act 2021 (
+        <strong>&quot;DPA&quot;</strong>) and general Privacy Principles.
+      </p>
+      <br />
+      <p>
+        Please read this Policy carefully. By using, accessing, connecting to,
+        or downloading any of the Service, you agree and consent to the
+        collection, use, protection, and disclosure of your information as
+        described in this Policy. If you do not agree to this Policy, please do
+        not use, access, connect to, or download any of the Service.
+      </p>
+      <br />
+      <h3>Personal Information We Collect</h3>
+      <br />
+      <p>We may collect the following categories of personal information:</p>
+      <ol>
```

Comment:
doc uses uppercase letters for the `<ol>`

Reply:
Yep, but in Terms doc also uses the mix of capitalize and uppercase headers. I chose the formatting we have in Terms.

Reply:
ok

Reply:
Done